### DISCUSSION TITLE
Naming react website url in github pages

### DISCUSSION THREAD

**Discussion by Shubhayu-Das** [community/community#21679] [Other Feature Feedback, Questions, & Ideas]
<p>While deploying my create-react-app website to Github pages, one of the steps involves setting up the homepage in package.json . </p>
<pre><code>{
...
"homepage": "https;//GithubUsername.github.io/reponame",
....
}
</code></pre>
<p> However, is it possible to set this to only <a href="http://GithubUsername.github.io/" rel="nofollow noopener">http://GithubUsername.github.io/</a> ?</p>

**Comment by lee-dohm** [comment]
<p>See <a href="https://help.github.com/en/articles/securing-your-github-pages-site-with-https" rel="nofollow noopener">our help docs on how to set up HTTPS support</a> for your GitHub Pages site.</p>
<p>Let us know if you have any questions.</p>

**Comment by Shubhayu-Das** [comment, marked as answer]
<p>Not what I was looking for😅. The answer to my question is to deploy on the “master” branch, with the name of the repository as <a href="http://username.github.io" rel="nofollow noopener">username.github.io</a></p>
